shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -940,6 +957,24 @@ - (BOOL)isAlwaysUse24HourFormat {
   return [dateFormat rangeOfString:@"a"].location == NSNotFound;
 }
 
+- (NSString*)brightnessMode {
+#if defined(__IPHONE_13_0)
+  if (@available(iOS 13, *)) {
+    UIUserInterfaceStyle style = UITraitCollection.currentTraitCollection.userInterfaceStyle;
```

Comment:
Why aren't you just using `self.traitCollection` here? That would avoid the SDK problem, right?

Reply:
I'll give that a try